src/stripe/stripe-service.ts

```diff
@@ -170,6 +171,11 @@ export class StripeService {
       layout: {
         type: "tabs",
       },
+      fields: {
+        billingDetails: {
+          address: "auto",
+        },
+      },
```

Comment:
Since this is the default value, I believe we shouldn’t need to set it explicitly